app/Anime.php

```diff
@@ -381,13 +381,17 @@ public function getSearchIndexSortBy(): array|null
     {
         return [
             [
-                "field" => "_text_match",
+                "field" => "_text_match(buckets:" . max_results_per_page() . ")",
                 "direction" => "desc"
             ],
             [
-                "field" => "members",
-                "direction" => "desc"
+                "field" => "popularity",
```

Comment:
Popularity is based on members. So we could just use that

Reply:
this one gives better results tho

Reply:
Huh interesting